packages/react-ai/src/components/AIConversation/views/default/Form.tsx

```diff
@@ -107,8 +107,17 @@ export const Form: Required<ControlsContextProps>['Form'] = ({
           rows={1}
           value={input?.text ?? ''}
           testId="text-input"
-          onCompositionStart={() => setComposing(true)}
-          onCompositionEnd={() => setComposing(false)}
+          onCompositionStart={() => {
+            setComposing(true);
+          }}
+          onCompositionEnd={(e) => {
+            const composedText = e?.currentTarget?.value || '';
+            setComposing(false);
+            setInput?.((prevValue) => ({
+              ...prevValue,
+              text: composedText,
+            }));
```

Comment:
What is the shape of `prevValue`? Tryng to understand the need for spreading the previous state

Reply:
My understanding is that the interface used for `prevValue` here is [ConversationInput](https://github.com/aws-amplify/amplify-ui/blob/326dfd6928d3273ac95ef5d088946f57afe6f5de/packages/react-ai/src/components/AIConversation/context/ConversationInputContext.tsx#L3) which only have `text` or `files` used to keep in track the data user previously composed.

Reply:
Got it, thanks!